web/src/HUD.tsx

```diff
@@ -139,6 +140,30 @@ class HUD extends Component<HudProps, HudState> {
     return this.pathBuilder.path(relPath)
   }
 
+  keymap() {
+    return {
+      clearSnackRestarts: "ctrl+shift+9",
+    }
+  }
+
+  handlers() {
+    return {
+      clearSnackRestarts: (event: KeyboardEvent | undefined) => {
+        console.log("Clearing snack restarts!")
+        debugger
+        if (this.state.View) {
+          this.state.View.Resources.forEach(r => {
+            fetch(
+              `//${window.location.host}/api/control/reset_restarts?name=${
```

Comment:
let's try to use path builder for constructing all URLs?

relatedly, i wonder if there should be a client-side-only way to hide restarts for sail.